Buildings/ThermalZones/EnergyPlus/Examples/VAVReheatRefBldgSmallOffice/BaseClasses/Floor.mo

```diff
@@ -139,25 +139,25 @@ model Floor "Model of a floor of the building"
         iconTransformation(extent={{380,150},{400,170}})));
   Airflow.Multizone.DoorDiscretizedOpen opeSouCor(
     redeclare package Medium = Medium,
-    wOpe=10,
+    wOpe=10*(17.69 + 8.46)/(40.76 + 24.13),
```

Comment:
What is the rationale for setting `wOpe` to this expression, which evaluates for all four openings to 4.03 m width. This seems small, I think the dividing wall between core zone and the other four zones is much longer, which is why I set it to 10 m length.

Reply:
I agree it is a bit small. The rationale is copied over from edits you made to the replaceable zone version, where you essentially scale the small office building sum of zone long and short wall with the sum of the same lengths for the larger building we used in the VAVReheat.

Reply:
@mwetter Let me know what would be your preference.

Reply:
I suggest you make the openings about half the length of the wall, and add a comment so we know in a few years where the number came from.

Reply:
Sounds good.